_targets.R

```diff
@@ -20,23 +40,15 @@ p1_targets_list <- list(
 
 p2_targets_list <- list(
   tar_target(
-    site_data_clean, 
-    process_data(site_data)
-  ),
-  tar_target(
-    site_data_annotated,
-    annotate_data(site_data_clean, site_filename = site_info_csv)
-  ),
-  tar_target(
-    site_data_styled,
-    style_data(site_data_annotated)
+    site_data_processed, 
+    process_data(filein = site_data, site_filename = site_info_csv)
```

Comment:
I think instead of `filein`, you want `input_data` (or you could go back to your `nwis_data`). This is because `site_data` is not a file, it is an R object. Meaning, you don't need to use `read_csv()` or anything like that in order to use it. `site_data` is ready to be used as-is. `filein` implies that the first step is to `read_csv()` and that `site_data` contains the character string of the filepath. 

I am going to use the _suggestions_ GitHub feature to suggest this change (since it is the only change I think you need to make). This will give you a chance to try it out. Essentially, it's a handy way for a reviewer to make a small edit directly to the code (very useful if you see typos) rather than describe the necessary edit which can speed up the process for very small, straightforward changes. You as the reviewer can either accept them (by clicking "commit") or reject the changes. If there is more than one suggestion (which there will be 3 related to this change), you can selected "Add suggestion to batch" in order to group the changes into a single commit.

Reply:
```suggestion
    process_data(nwis_data = site_data, site_filename = site_info_csv)
```